core/src/main/java/org/kohsuke/stapler/export/ModelBuilder.java

```diff
@@ -42,12 +42,22 @@
     /*package*/ final Map<Class, Model> models = new ConcurrentHashMap<Class, Model>();
 
     public <T> Model<T> get(Class<T> type) throws NotExportableException {
+        if (type.getAnnotation(ExportedBean.class) == null) {
+            throw new NotExportableException(type);
+        }
         return get(type, null, null);
     }
 
-    public <T> Model<T> get(Class<T> type, @CheckForNull Class<?> propertyOwner, @Nullable String property) throws NotExportableException {
+    public <T> Model<T> get(Class<T> type, @CheckForNull Class<?> propertyOwner, @Nullable String property) {
+        if (type.getAnnotation(ExportedBean.class) == null) {
+            throw new NotExportableException(type);
+        }
+        return getOrNull(type, propertyOwner, property);
+    }
+
+    public <T> Model<T> getOrNull(Class<T> type, @CheckForNull Class<?> propertyOwner, @Nullable String property) throws NotExportableException {
```

Comment:
`@CheckForNull` return. And why is this declared to throw `NotExportableException`?